e2e/commands/start/start_test.go

```diff
@@ -17,28 +17,36 @@ package start_test
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 
+	"github.com/ZupIT/horusec/internal/utils/testutil"
 	"github.com/google/uuid"
+	"github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gexec"
 
 	"github.com/ZupIT/horusec-devkit/pkg/utils/logger/enums"
-
-	"path/filepath"
-
-	"github.com/ZupIT/horusec/internal/utils/testutil"
 )
 
 var _ = Describe("running binary Horusec with start parameter", func() {
 	var (
-		session           *gexec.Session
-		err               error
-		flags             map[string]string
-		repoAuthorization string
-		configFilePath    = testutil.GoExample1
+		session                 *gexec.Session
+		err                     error
+		flags                   map[string]string
+		repoAuthorization       string
+		projectPath             = testutil.GoExample1
+		certificateFileWithPath string
 	)
 
+	BeforeSuite(func() {
+		file, err := os.CreateTemp(os.TempDir(), "*.crt")
+		if err != nil {
+			ginkgo.Fail(fmt.Sprintf("error: %v", err))
```

Comment:
Use just `Fail(fmt.Sprintf("error: %v", err))`
ginko is already imported using `.`

Reply:
adjusted!